server/datastore/mysql/hosts.go

```diff
@@ -2227,7 +2227,7 @@ func (ds *Datastore) FailingPoliciesCount(ctx context.Context, host *fleet.Host)
 	}
 
 	query := `
-		SELECT SUM(1 - pm.passes) AS n_failed
+		SELECT SUM(1 - COALESCE(pm.passes, 0)) AS n_failed
 		FROM policy_membership pm
 		WHERE pm.host_id = ?
 		GROUP BY host_id
```

Comment:
See @zhumo's [proposal](https://github.com/fleetdm/fleet/pull/12523#issuecomment-1615316499), maybe we can ignore these "pending" policies in the count by using `WHERE pm.passes IS NOT null`?